doc/whats_new.rst

```diff
@@ -100,6 +100,10 @@ Enhancements
     - Extensive plotting example shows how to use contours and filled contours
       on glass brain.
 
+    - Function :func:`nilearn._utils.ndimage.largest_connected_component_img`
```

Comment:
Would this mean adding 
`from .ndimage import largest_connected_component_img`  
and `'largest_connected_component_img'` to `__all__` in `_utils/__init__.py`?

(I'll do all the little changes later -- and I thought I spend extra attention to the little details :/ . I have flake8 installed though, but I am also quite new to vim - so apparently my setup still isn't working as expected.)